### ISSUE TITLE
ones a problem and one is a request.

### ISSUE THREAD

**Issue by XDeadcrow12** [TheGameCreators/GameGuruRepo#3367]
this could be something I'm doing wrong but I am not %100 sure what is going on.
I have multiple projects going but sometimes even after I save and quit 
my level is completely gone. it would be different I had not saved it
but it will be there then disappear and not resurface. 

the request Is this, I know the staff are working very hard and trying to fix things
and get things done but I wondering is there an estimated time on when we will
see Third person come into play ?.

**Comment by MonkeyFrogStudio** [comment]
@XDeadcrow12 From what's been said, 3rd person is not going to be added to GameGuru MAX at all. At least it's not in the plans. That doesn't prevent you or someone else from programming a third-person camera. I believe someone had done that for GameGuru Classic (though I have no idea how well it worked). So, I wouldn't wait on TGC adding third-person to MAX. 

As far as your first issue, I'm not sure what's going on there. I have several projects going on and never have lost a level. Have you tried to look for it? Your levels are saved here:

Documents>GameGuruApps>GameGuruMAX>files>mapbank

Try going there and looking to see if your level is there by whatever name you'd saved it as. In the editor, you can always open a level, if you find it there.

In the future, it's best to post multiple things (multiple bugs, multiple requests, or any combination) each in their own post. This way TGC can assign each one to a different individual, if need be, to be taken care of. Also, it makes it possible to solve and close one without closing the other (if it's not been resolved), etc. When more than one subject is posted in a single post, it makes it a lot harder on TGC to track things, assign it to specific developers, etc.

**Comment by davetgc** [comment]
Please list bugs and requests as separate reports. Thanks,

**Comment by XDeadcrow12** [comment]
Yeah Third person in GG Classic really sucked. then again i know it wasn't
really meant for it so i get it.

Thank you for the information and your time.

I will try and search for it again.

Regards.
